### ISSUE TITLE
Question regarding trainer.py

### ISSUE THREAD

**Issue by JamiePW** [vfeofanov/mantis#4]
I want to obtain the logits of the model's predicted output, so I modified the code on line 263 of `/src/mantis/trainer/trainer.py` to `return probs`. However, the shape of the output after running the code is still `(n_samples,)`.

**Comment by JamiePW** [comment]
we need to use `from src.mantis.trainer.trainer import MantisTrainer`
